### ISSUE TITLE
Describing Expressiveness of Ontologies

### ISSUE THREAD

**Issue by AleSteB** [nfdi4cat/Ontology-Overview-of-NFDI4Cat#4]
Classification of ontologies with regards to expressivity (OWL2 EL/QL/RL)
https://www.w3.org/TR/owl2-profiles/#Introduction

OWL2 EL - for ontologies that contain very large numbers of properties and/or classes 

OWL2 QL - for ontologies that use very large volumes of instance data, and where query answering is the most important reasoning task.

OWL2 RL - for ontologies that require scalable reasoning without sacrificing too much expressive power.

**Comment by AleSteB** [comment]
Also further classification reasonable regarding domain ontology VS application ontology etc?
But hard criteria for this to be looked up

**Comment by AleSteB** [comment]
The I-ADOPT Ontology-Classes could be reused for our classification of the ontologies!

https://i-adopt.github.io/

**Comment by HendrikBorgelt** [comment]
I-adopt has very few classes with constrain, entity, variable, variable set and property, therfore i do not think that classifying the ontologies with these terms will be sufficient.

**Comment by AleSteB** [comment]
Yes, I agree! However, we might use it anyway and extend the classifications of it to obtain an "ontology about ontologies"?

**Comment by AleSteB** [comment]
As [StroemPhi](https://github.com/StroemPhi) explained in Slack, MOD and/or OMO could also be promising ontologies for this purpose.
